data/saas/config/powerreviews_config.yml

```
@@ -0,0 +1,60 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: PowerReviews
+  type: powerreviews
+  description: A sample schema representing the PowerReviews connector for Fides
+  version: 0.1.0
+
+  connector_params:
+    - name: domain
+      default_value: enterprise-api.powerreviews.com
+      description: The base URL for the PowerReviews API
+    - name: client_id
+      description: Your PowerReviews client ID
+      label: Client ID
+    - name: client_secret
+      description: Your PowerReviews client secret
+      label: Client secret
+      sensitive: True
+
+  client_config:
+    protocol: https
+    host: <domain>
+    authentication:
+      strategy: oauth2_client_credentials
+      configuration:
+        token_request:
+          method: POST
+          path: /oauth2/token
+          headers:
+            - name: Content-Type
+              value: application/x-www-form-urlencoded
+          client_config:
+            protocol: https
+            host: <domain>
+            authentication:
+              strategy: basic
+              configuration:
+                username: <client_id>
+                password: <client_secret>
+          body: |
+            {
+              "grant_type": "client_credentials"
+            }
+
+
+  test_request:
+    method: GET
+    path: /v1/reviews
+    query_params:
+      - name: limit
+        value: 3
+
+  endpoints:
+    - name: privacy
```

Let's rename this to `user` so it reflects the type of object being deleted
```suggestion
    - name: user
```